z3c/dependencychecker/main.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from z3c.dependencychecker.dependencychecker import logger
+from z3c.dependencychecker.dependencychecker import parse_command_line
+from z3c.dependencychecker.dependencychecker import set_log_level
+
+import os
+import pkg_resources
+import sys
+
+
+def main():
+    options, args = parse_command_line()
+    set_log_level(options.verbose)
+    path = get_path(args)
+    get_package_dependencies(path)
+
+
+def get_path(args):
+    """Get path from given arguments or use """
+    path = os.getcwd()
+    if len(args) > 0:
+        path = os.path.abspath(args[0])
+        if not os.path.isdir(path):
+            logger.info('Given path does not exist: {0}'.format(args[0]))
+            sys.exit(1)
+    logger.info('path used: %s', path)
+    return path
+
+
+def get_package_dependencies(path):
+    if not has_setup_py_file(path):
+        logger.info('No setup.py file found! on {0}'.format(path))
+        sys.exit(1)
+
+    new_path, filename = get_egg_info_folder(path)
+    if not new_path:
+        logger.info('No egg-info folder found! on {0}'.format(new_path))
```

Comment:
http://reinout.vanrees.org/weblog/2015/06/05/logging-formatting.html

So the best way is to stick to ye olde `%s` :-)

Reply:
Yup, I will fix that up.